docs/installation.md

```diff
@@ -15,6 +16,9 @@ First, add our official ralph repository:
     sudo apt-key adv --keyserver  hkp://keyserver.ubuntu.com:80 --recv-keys 379CE192D401AB61
     sudo sh -c "echo 'deb https://dl.bintray.com/vi4m/ralph wheezy main' >  /etc/apt/sources.list.d/vi4m_ralph.list"
 
+After, install the secure transport for accessing repository :
```

Comment:
yes - `apt-get update` definitely shouldn't be executed before installing https transport. @cdb-fr - we've made some changed to the docs recently - could you apply change from this conversation and make it mergable?